docs/modules/ROOT/pages/reference/extensions/jackson.adoc

```diff
@@ -67,18 +77,30 @@ The Quarkus Jackson extension exposes an `ObjectMapper` CDI bean which can be di
 
 [source,java]
 ----
-ObjectMapper mapper = new ObjectMapper();
-JacksonDataFormat dataFormat = new JacksonDataFormat();
-// Make JacksonDataFormat discover the Quarkus Jackson `ObjectMapper` from the Camel registry
-dataFormat.setAutoDiscoverObjectMapper(true);
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.jackson.JacksonDataFormat;
+
+public class Routes extends RouteBuilder {
+    public void configure() {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
My mistake here. You don't actually need to instantiate `ObjectMapper` in this scenario because `JacksonDataFormat` will resolve the Quarkus one from the Camel registry.

Reply:
Thanks, done.